src/Sulu/Bundle/AdminBundle/Resources/js/views/List/toolbarActions/UploadToolbarAction.js

```diff
@@ -0,0 +1,307 @@
+        }
+
+        return symfonyRouting.generate(routeName, this.requestParameters);
+    }
+
+    @computed get errorMapping(): $ReadOnly<Object> {
+        const {errorMapping = {}} = this.options;
+
+        if (typeof errorMapping !== 'object') {
+            throw new Error('The "errorMapping" option must be an object!');
+        }
+
+        return errorMapping;
+    }
+
+    @computed get requestParameters(): $ReadOnly<Object> {
+        const {
+            options: {
+                requestParameters: attributesToRequest = {},
+                routerAttributesToRequest = {},
+            },
+            router: {
+                attributes: routerAttributes,
+            },
+        } = this;
+
+        if (!attributesToRequest || typeof attributesToRequest !== 'object') {
+            throw new Error('The "attributesToRequest" option must be an object!');
+        }
+
+        if (!routerAttributesToRequest || typeof routerAttributesToRequest !== 'object') {
+            throw new Error('The "routerAttributesToRequest" option must be an object!');
+        }
+
+        let requestParameters = {};
+        Object.keys(routerAttributesToRequest)
+            .forEach((routerAttributeKey) => {
+                const requestAttributeKey = routerAttributesToRequest[routerAttributeKey];
+
+                if (typeof requestAttributeKey !== 'string') {
+                    throw new Error('The values "routerAttributesToRequest" must be strings!');
+                }
+
+                const attributeName = isNaN(routerAttributeKey)
+                    ? routerAttributeKey
+                    : requestAttributeKey;
+
+                requestParameters[requestAttributeKey] = routerAttributes[attributeName];
+            });
+        requestParameters = {...requestParameters, ...attributesToRequest};
+
+        return requestParameters;
+    }
+
+    @computed get accept(): ?$ReadOnlyArray<any> {
+        const {accept} = this.options;
+
+        if (accept === undefined || accept === null) {
+            return undefined;
+        }
+
+        if (!Array.isArray(accept)) {
+            throw new Error('The "accept" option must be an array!');
+        }
+
+        return accept;
+    }
+
+    @computed get minSize(): ?number {
+        const {minSize} = this.options;
+
+        if (minSize === undefined || minSize === null) {
+            return undefined;
+        }
+
+        if (typeof minSize !== 'number') {
+            throw new Error('The "minSize" option must be a number!');
+        }
+
+        return minSize;
+    }
+
+    @computed get maxSize(): ?number {
+        const {maxSize} = this.options;
+
+        if (maxSize === undefined || maxSize === null) {
+            return undefined;
+        }
+
+        if (typeof maxSize !== 'number') {
+            throw new Error('The "maxSize" option must be a number!');
+        }
+
+        return maxSize;
+    }
+
+    @computed get multiple(): boolean {
```

Comment:
think i would call this `allowMultiple`, but dont have a strong opinion.

Reply:
Since the `multiple` prop of a html input field is also just called `multiple`, I would prefer that.

Reply:
did not know that - is okay 👍